core/domain/suggestion_services.py

```diff
@@ -1003,6 +1006,11 @@ def get_reviewable_translation_suggestions_by_offset(
         user_id)
     language_codes = (
         contribution_rights.can_review_translation_for_language_codes)
+
+    if language:
+        language_codes = [l for l in language_codes
+                         if l == language]
```

Comment:
```suggestion
    if language is not None and language in language_codes:
        language_codes = [language]
```
Seems more explicit.

Reply:
Turns out that this suggested change broke the backend tests. I re-implemented the original change without list comprehension, in a more readable way. Hopefully it is clearer now.